README.md

```diff
@@ -1 +1,61 @@
-# Awesome-Books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Awesome-Books
+
+
+
+> A dynamic collection of books using JavaScript.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Getting Started
+
+**To set up, clone the project on your local system, and run it using a web browser.**
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```

Comment:
[ Optional ]

For headings that you do not have content, you can remove them so the README looks neat 👍